app/src/main/java/com/h/pixeldroid/objects/Post.kt

```diff
@@ -0,0 +1,37 @@
+package com.h.pixeldroid.objects
+
+import java.io.Serializable
+
+class Post(private val status: Status?) : Serializable {
+    private val defaultAvatar = "https://raw.githubusercontent.com/pixelfed/pixelfed/dev/public/img/pixelfed-icon-color.png"
```

Comment:
Probably shouldn't use a link to an image on github here, and this file should not be in the objects package (it is not an object generated from the Pixelfed API)